### ISSUE TITLE
Missing luma/__init__.py when installing with "setup.py install"

### ISSUE THREAD

**Issue by Gadgetoid** [rm-hull/luma.core#144]
I cloned luma.core onto a Pi 3B+ running Raspbian Stretch Lite and built/installed with "sudo python setup.py install" (setuptools 33.1.1) but receive an import error when trying to import the installed package.

* `git clone https://github.com/rm-hull/luma.core`
* `cd luma.core`
* `sudo python setup.py install`
* `cd ~`
*  `python` and `import luma.core` - "No module named luma.core"

The output of `pip show luma.core` shows it's installed in `/usr/local/lib/python2.7/dist-packages/luma.core-1.7.2-py2.7.egg` as I would expect.

Upon poking into this directory I find that the `luma` directory does not contain the `__init__.py` that it should, meaning it's not recognised as a namespace package.

This file is, however, present in the source `luma` directory and is a pkgutil style namespace package.

However you seem to be declaring this as a pkg_resources style namespace package with the `namespace_packages=['luma']` line in `setup.py`. This looks like a conflation of both techniques.

The `namespace_packages` argument should be dropped, and the `luma` package added to `packages=[]`.

luma.oled has this same issue.

See: https://packaging.python.org/guides/packaging-namespace-packages/#pkgutil-style-namespace-packages - which  links to this example project: https://github.com/pypa/sample-namespace-packages/blob/master/pkgutil/pkg_a/setup.py

The example project uses `find_packages()` which adds both the main package and the namespace package to the `packages` argument.

pkg_resources style packages - compatible with only Python 3.x (afaik) - declare the package in *both* the `packages` and `namespace_packages` arguments but use a slightly different style of `__init__.py` which contains only: `__import__('pkg_resources').declare_namespace(__name__)`.

TLDR: you need to remove `namespace_packages=["luma"]` from `setup.py` and simply add `"luma"` into `packages=[]`
